components/org.wso2.carbon.utils/src/main/java/org/wso2/carbon/utils/Constants.java

```diff
@@ -0,0 +1,70 @@
+/*
+ *  Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Since this is an existing class and I didn't do any changes hence didn't touch the year